web/styles/settings.css

```diff
@@ -1937,3 +1937,34 @@ $option_title_width: 180px;
         opacity: 0.7;
     }
 }
+
+.pill_container {
+    display: -webkit-box;
+    -webkit-line-clamp: 1;
+    -webkit-box-orient: block-axis;
```

Comment:
I can see that some of these properties are non-standard and may have some issues [here[(https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp). Can we instead use flex layout? If there is no other way to make this work as expected, as we already use these properties in a couple of places, but we can try once.

Reply:
Changed this to use flex layout.